src/test/java/com/puppycrawl/tools/checkstyle/utils/AnnotationUtilTest.java

```diff
@@ -61,6 +61,13 @@ public void testContainsAnnotationNull() {
         }
     }
 
+    @Test
+    public void testHasOverrideAnnotationNull() {
+        assertWithMessage("Should be false for null ast")
```

Comment:
Can we make this test to be Input based ?

Reply:
As we do simple refactoring, we should have already all Inputs with all edgy cases

Reply:
Pitest codding is failing

Reply:
removed. Pitest failure was random, there was no changes in check that failed. Now it is ok